src/Orchard.Web/Modules/Orchard.ImportExport/Services/SetupService.cs

```diff
@@ -101,6 +102,10 @@ public class SetupService : Component, ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
```

Comment:
He's not me yet, but I recognize a few bits and bobs that remind me of myself :)